packages/common/src/types/Aliases.ts

```diff
@@ -18,6 +18,11 @@ export type SemVerString = string
  * TODO: make ping filtering sane!
  */
 export type PingFilter = (pingResponse: PingResponse, gsnTransactionDetails: GsnTransactionDetails) => void
+
+/**
```

Comment:
comment is misplaced: its not a definition of the type, but of the usage, for asyncPaymasterData callback